ethdb/leveldb/leveldb.go

```diff
@@ -100,6 +107,22 @@ func New(file string, cache int, handles int, namespace string, readonly bool) (
 		options.BlockCacheCapacity = cache / 2 * opt.MiB
 		options.WriteBuffer = cache / 4 * opt.MiB // Two of these are used internally
 
+		if config.CompactionTableSize != 0 {
+			options.CompactionTableSize = int(config.CompactionTableSize * opt.MiB)
```

Comment:
The gist of this PR is to set these LevelDB configs.

The rest is just plumbing to convey the CLI flag values (if any) through the pipes to this method.